app/models/document.rb

```diff
@@ -1,7 +1,8 @@
 class Document < ApplicationRecord
   belongs_to :application
 
-  has_attached_file :attachment, PAPERCLIP_STORAGE_OPTIONS 
+  has_attached_file :attachment
+  # , PAPERCLIP_STORAGE_OPTIONS 
```

Comment:
Don't uncomment this - put back as it was. Go to config/environments/test.rb and add `PAPERCLIP_STORAGE_OPTIONS = {}`